filter/filtering.go

```diff
@@ -0,0 +1,67 @@
+// Package filter provides filter model
+package filter
+
+import (
+	"database/sql"
+
+	"github.com/google/uuid"
+)
+
+type Postgres struct {
+	db *sql.DB
+}
+
+// NewPostgres returns Postgres with db
+func NewPostgres(db *sql.DB) *Postgres {
+	return &Postgres{db: db}
+}
+
+// AddFilter adds new filter to database
+func (p *Postgres) AddFilter(filter *Filter) error {
+	sqlStatement := "INSERT INTO FILTERS(name,type,configutation,reversed,user_id) VALUES ($1,$2,$3,$4,$5)"

```

Comment:
WDYT about moving all queries strings into constants block? _E.g. addQuery, getQuery, updateQuery and deleteQuery_

Reply:
Oh, seems to be effective)

Reply:
fixed